charts/images.yaml

```diff
@@ -111,8 +111,14 @@ images:
   tag: v0.56.2
 - name: kube-state-metrics
   sourceRepository: github.com/kubernetes/kube-state-metrics
-  repository: quay.io/coreos/kube-state-metrics
-  tag: v1.9.7
+  repository: registry.k8s.io/kube-state-metrics/kube-state-metrics
```

Comment:
Can you apply the following diff with cosmetics improvements only?

```diff
diff --git a/charts/images.yaml b/charts/images.yaml
index 5e0d7114f..640c7ddef 100644
--- a/charts/images.yaml
+++ b/charts/images.yaml
@@ -112,13 +112,13 @@ images:
 - name: kube-state-metrics
   sourceRepository: github.com/kubernetes/kube-state-metrics
   repository: registry.k8s.io/kube-state-metrics/kube-state-metrics
-  tag: "v2.5.0"
-  targetVersion: ">= 1.20"
+  tag: v2.1.1
+  targetVersion: "< 1.20"
 - name: kube-state-metrics
   sourceRepository: github.com/kubernetes/kube-state-metrics
   repository: registry.k8s.io/kube-state-metrics/kube-state-metrics
-  tag: v2.1.1
-  targetVersion: "< 1.20"
+  tag: v2.5.0
+  targetVersion: ">= 1.20"
 - name: node-exporter
   sourceRepository: github.com/prometheus/node_exporter
   repository: quay.io/prometheus/node-exporter
```

Makes v2.1.1 the fist entry and removes the quotes from the tag for consistency reasons.

Reply:
👍 -> https://github.com/gardener/gardener/pull/6224/commits/37d30537368366a6af3de526310ddd4ae68ecd22